apps/registry/app/[username]/decisions/hooks/useDecisionTree.js

```
@@ -0,0 +1,529 @@
+      } catch (error) {
+        console.error('AI evaluation error:', error);
+        setMatchResult({
+          outcome: 'error',
+          bucket: '⚠️ Evaluation Error',
+          reasons: [['Error', error.message]],
+          score: 0,
+        });
+      }
+    },
+    [resetHighlights, preferences, animateAIPath]
+  );
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)